src/main/java/com/google/sps/servlets/AboutServlet.java

```diff
@@ -49,7 +50,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     Map<String, Object> context = new HashMap<>();
-    context.put("url", "/");
+    context.put(URLPatterns.URL, URLPatterns.ABOUT);
```

Comment:
Typo? This does not seem to be ABOUT

Reply:
the old value was incorrect. "/" should have been "/about", it was not updated correctly when we first made the aboutServlet.java